### DISCUSSION TITLE
Delete All Photos in Google Photos Library?

### DISCUSSION THREAD

**Discussion by PerfectCr** [TheLastGimbus/GooglePhotosTakeoutHelper#234] [General]

**Comment by michalecm** [comment]
You can do this easily from Account Data.

**Reply by michalecm** [reply]
accounts.google.com -> click 'data & privacy' in left menu -> scroll down to 'Apps & Services' -> click on 'A summary of your services and data' -> delete a service -> delete a service -> delete google photos (or for me it shows google one, but I don't have anything in drive)

**Comment by treborsitnay** [comment]
TheLastGimbus, do you by chance have a similar script/app to do the same merge for Gmail (with or without attachments), Contacts, etc? or know of any such apps/scripts?
-rdy-or-not
